src/service/repo/signedMessageRepo.ts

```diff
@@ -12,6 +12,7 @@ type SendPayloadType = {
   message: string;
   channelType: number;
   messageType?: string;
+  attachments?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` property is of type `any`. It's a good practice to avoid using `any` as it can lead to potential runtime errors. If possible, define an interface for the attachments.
```diff
-  attachments?: any;
+  attachments?: AttachmentType[];
```
Where `AttachmentType` could be an interface that describes the structure of an attachment.



<!-- This is an auto-generated comment by OSS CodeRabbit -->